src/main.cpp

```diff
@@ -119,6 +133,9 @@ int main() {
 
     shardThread = new std::thread(startShard, (CNShardServer*)shardServer);
 
+    displayLogo();
+    std::cout << "[INFO] Server is up and ready, please enjoy!\n";
+
     loginServer.start();
```

Comment:
1. The message shouldn't display until _after_ the login server has been started.
2. Use `std::endl` instead of `\n` since that's what's being used on line 130.
3. The tone of the line doesn't fit well with the rest of the server output. We don't need to user-face _that hard._

```suggestion
    loginServer.start();
    
    std::cout << "[INFO] Server is up." << std::endl;
```